pkg/server/endpoints/bundle/internal/autocert/autocert.go

```diff
@@ -1199,6 +1212,17 @@ func (r *lockedMathRand) int63n(max int64) int64 {
 	return n
 }
 
+func supportedSignatureAlgorithms(privKey crypto.Signer) []tls.SignatureScheme {
+	var out []tls.SignatureScheme
+	switch privKey.Public().(type) {
+	case *ecdsa.PublicKey:
```

Comment:
I wonder if we are being too restrictive here.

For example, AWS KMS supports EC P-384 keys and SHA384 (i.e. `tls.ECDSAWithP384AndSHA384`), and it should also be supported for TLS 1.3:
https://wiki.mozilla.org/Security/Server_Side_TLS#Intermediate_compatibility_.28recommended.29

Reply:
Our ACME layer only uses RSA2048 and ECP256 keys...

Reply:
Ah, right. Thanks for reminding me.
https://github.com/spiffe/spire/blob/2632bb31dfd8b8ae5e59f2cd723c78655678093f/pkg/server/endpoints/bundle/acme_auth.go#L120-L123